test/test_suit/test_schema_mapping.py

```diff
@@ -0,0 +1,76 @@
+from test_suit.test_utils import RetryableError, NonRetryableError
+import json
+
+# test if each type of data fit into the right column with the right type
+# also test if the metadata column is automatically added
+class TestSchemaMapping:
```

Comment:
if the kafka message has two columns c1 and C1, will we be able to mapped to the current column in the table?

Reply:
I believe they will be mapped into the same column C1 with the current behavior. I think we could add an option in the sf config to allow uppercase letter, and we can add the quotation marks in the connector.

Reply:
Are you saying the second value will override the first value?

Reply:
I suppose so. I think this happens in the SDK.

Reply:
Could we add a verification step in KC to check whether there're two columns with the same name if ignoring cases, and fail saying we can't supported?

Reply:
Just added the check and tested it. I could add a UT if you think it's necessary